logstash-core/src/main/java/org/logstash/log/StructuredMessage.java

```diff
@@ -0,0 +1,93 @@
+package org.logstash.log;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.apache.logging.log4j.message.Message;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+@JsonSerialize(using = CustomLogEventSerializer.class)
```

Comment:
It seems that annotation `@JsonSerialize(using = CustomLogEventSerializer.class) ` is not useful.